module/database/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "zone" {
+  type = string
+}
+
+variable "region" {
+  type = string
+}
+
+variable "env" {
+  type = string
+}
+
+variable "rdb_is_ha_cluster" {
+  type = bool
+}
+
+variable "rdb_disable_backup" {
+  type = bool
+}
+
+variable "rdb_instance_node_type" {
+  type = string
+}
+
+variable "rdb_instance_engine" {
+  type = string
```

Comment:
Is there any default?

Reply:
No, I specify this rdb_instance_engine = "PostgreSQL-13" in the main.tf